src/main/java/pokecube/core/client/PokecenterSound.java

```diff
@@ -36,7 +36,7 @@ public void tick()
         if (this.stopped)
         {
             this.volume = 0;
-            this.donePlaying = true;
+            this.repeat = true;
```

Comment:
This file needed to be entirely looked into, so I am not sure if this fix would work.